patronload/patron.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import re
+from datetime import date
+from typing import Any, Generator
+
+from bs4 import BeautifulSoup
+from dateutil.relativedelta import relativedelta
+
+logger = logging.getLogger(__name__)
+
+
+def create_patron_dicts(
```

Comment:
I think you can do away with this function and just use`dict(zip(fields, patron_record))` wherever you need to generate the dict.